crates/containerd-shim-wasmedge/src/instance.rs

```diff
@@ -13,6 +8,11 @@ use containerd_shim_wasm::sandbox::instance_utils::maybe_open_stdio;
 use containerd_shim_wasm::sandbox::{EngineGetter, InstanceConfig};
 use nix::unistd::close;
 use serde::{Deserialize, Serialize};
+use std::fs::File;
```

Comment:
Why are these imports moved to here? Did you use a formatter to do it?

Reply:
It think so, I didn't notice but I did run `make fix` which runs fmt and clippy and looks like it re-ordered them alphabetically

Reply:
umm rustfmt and clippy won't re-order imports alphabetically IIRC.